sympy/polys/ring_series.py

```diff
@@ -53,6 +53,7 @@
 from sympy.utilities.misc import as_int
 from mpmath.libmp.libintmath import giant_steps
 import math
+from sympy import Dummy
```

Comment:
Internal imports should use full qualification:
```
from sympy.core.symbol import Dummy
```

Reply:
Changed in dafc95faeb6d6a78e397d061455cda6218ca6017